src/content/docs/style-guide/writing-docs/writer-workflow/swiftype-search.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Swiftype search
+topics:
+  - Tech writer style guide
+  - Processes and procedures
+---
+
+
+import styleguideNewRelicUiQuestionMark from 'images/style-guide_screenshot-crop_new-relic-ui-question-mark.png'
+
+import styleguideUiSearchString from 'images/style-guide_screenshot-crop_ui-search-string.png'
+
+
+Swiftype is the elastic-based search engine that powers the search in `docs.newrelic.com` and the produdt UI.
+
+As a technical writer at New Relic, you may need to log into Swiftype and do a variety of tasks to make sure your docs are recognized during internal searches.
+
+## Where are Swiftype results used? [#where-used]
+
+A growing number of sites and projects use our Swiftype engine to provide search results:
+
+* The New Relic help center
+* Support landing page
+* Most non-product New Relic sites (developer for related content, docs, storefront, etc.)
+
+## Swiftype crawler info [#crawler-info]
+
+The crawler says it crawls every 12 hours. The rate at which it crawls is relatively slow, which means it could take longer than that to pick up page changes. It also uses some combination of `sitemap.xml` and link collection when crawling.
+
+We can trigger a manual crawl once every 12 hours, if needed. Or, we can have Swiftype crawl specific pages when needed.
+
+The average re-index time for docs is four days.
+
+## Manually add new page to Swiftype index [#manually-index]
+
+You may want to manually add a new page to the Swiftype index when the crawl is too slow and you want the page available right away. Keep in mind that updating the idex may produce quick results for the docs site, you may experience a lag for the docs in the product UI.
+
+To manually add a new page in Swiftype:
+
+1. Ensure the page you want indexed does not have meta robots noindex set and that it's available publicly.
```

Comment:
Do we need to say how or where to check for this?

Reply:
@stumblefiend I think your version of develop was a bit out of date because I had already replaced this step with another one. I'll go ahead and close this.